src/main/target/HAKRCF405/target.h

```diff
@@ -17,12 +17,13 @@
  *
  * If not, see <http://www.gnu.org/licenses/>.
  */
- 
 
 #pragma once
 
-#define TARGET_BOARD_IDENTIFIER "HKF4"
-#define USBD_PRODUCT_STRING  "HAKRCF405"
+#define TARGET_BOARD_IDENTIFIER "HRF7"
```

Comment:
This looks wrong.

Reply:
Sorry, I copy error code  to this github base code.